connector/src/main/java/org/geysermc/connector/GeyserConnector.java

```diff
@@ -141,13 +142,49 @@ public void shutdown() {
         bootstrap.getGeyserLogger().info("Shutting down Geyser.");
         shuttingDown = true;
 
+        if (players.size() >= 1) {
+            bootstrap.getGeyserLogger().info("Kicking " + (players.size() / 3) + " player(s)");
```

Comment:
If we refactor how player size is determined then we'll need to change this line.

Reply:
With https://github.com/GeyserMC/Geyser/commit/ade40d5a8b34e848b9c71d16427317e429bc6f07 this needs to be changed.